chemprop/cli/predict.py

```diff
@@ -56,6 +57,7 @@ def add_predict_args(parser: ArgumentParser) -> ArgumentParser:
     )
     parser.add_argument(
         "--model-path",
+        "--model-paths",
```

Comment:
I don't have a strong preference on whether to put `--model-path` or `--model-paths` first. Since we used `args.model-paths` in predict.py, maybe we should make this change. Then, we should also modify the documentation. I will wait for @KnathanM's comments on this before making any changes.

Reply:
I also don't have a strong preference for which to put first.  I think using `args.model_paths` is maybe better because it makes it easier to know that variable is a list. So that would be accepting Hao-Wei's suggestion.